theme/static/js/custom.js

```diff
@@ -143,6 +143,7 @@ $(document).ready(function () {
     for (var i = 0; i < keywords.length; i++) {
         if (keywords[i]) {
             var li = $("<li><a class='tag'></a></li>");
+            li.find('a').attr("href", "/search/?q=" + keywords[i]);
```

Comment:
Close. You can do one better with:

`li.find('a').attr("href", "/search/?q=&selected_facets=subject_exact:" + keywords[i]);`

See https://www.hydroshare.org/search/?q=&selected_facets=subject_exact:USACE%20Corps%20Water%20Management as an example